### PR TITLE
opx-config automation : phase 1

### PR DESCRIPTION
**Pull request by rakeshdatta**
* Replaced the ip address, static route, mac, VLAN and LAG old configuration commands with new opx-config-commands
* Added l3-basic-functionality, data_vrf_basic, vlanConfig, lldp and lagConfig test scripts.

### CODE REVIEW

**Comment by jeffyin-dell** [comment]
Approved, but require @madhu222 's approval before merge

**Comment by jeffyin-dell** [comment]
@madhu222 please review/approve?

**Comment by madhu222** [comment]
@rakeshdatta @Tejaswi-Goel Can you please run the sanity  and post the logs here?
@rakeshdatta In your opx-config-vlan and opx-config-lag, Can you please separate the TCs with block and clean up ? I m removing the sanity from your scripts for now. After you are done with the changes, we will include the sanity tag for the TCs

**Comment by rakeshdatta** [comment]
UPDATE: @Tejaswi-Goel is waiting for an available test bed for the sanity tests to be run.

**Review by madhu222 — CHANGES_REQUESTED** [review]
Can you please remove the spaces between tags ,always? There should not be any space in a block. Can you make changes in both opx-config-vlan and opx-config-lag

File: playbook/Forwarding/roles/span_mirroring/tasks/setup.yml
Comment (madhu222) [inline playbook/Forwarding/roles/span_mirroring/tasks/setup.yml]: Remove the commented ones here

File: playbook/L2/roles/vlanConfig/tasks/vlanConfig.yml
Comment (madhu222) [inline playbook/L2/roles/vlanConfig/tasks/vlanConfig.yml]: Remove the space here

File: playbook/L2/roles/vlanConfig/tasks/vlanConfig.yml
Comment (madhu222) [inline playbook/L2/roles/vlanConfig/tasks/vlanConfig.yml]: Remove the space

File: playbook/L2/roles/vlanConfig/tasks/vlanConfig.yml
Comment (madhu222) [inline playbook/L2/roles/vlanConfig/tasks/vlanConfig.yml]: Remove the space

File: playbook/L2/roles/vlanConfig/tasks/vlanConfig.yml
Comment (madhu222) [inline playbook/L2/roles/vlanConfig/tasks/vlanConfig.yml]: Remove the space

File: playbook/L2/roles/vlanConfig/tasks/vlanConfig.yml
Comment (madhu222) [inline playbook/L2/roles/vlanConfig/tasks/vlanConfig.yml:66]: Remove the space,

**Review by madhu222 — CHANGES_REQUESTED** [review]
I have mentioned where to remove the space in one test case. Please do it for all the test cases. And can you please remove the sanity from the failed test case for now?

File: playbook/Forwarding/roles/span_mirroring/tasks/setup.yml
Comment (rakeshdatta) [reply playbook/Forwarding/roles/span_mirroring/tasks/setup.yml]: Not my change.

**Review by madhu222 — APPROVED** [review]
Looks good to me.

**Comment by jeff-yin** [comment]
Needs DCO signoff...

**Review by jeffyin-dell — CHANGES_REQUESTED** [review]
Commit needs sign-off

**Pull request closed without merging**
